security_monitoring/assets/saved_views/default_signal_view.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "Security Monitoring (Default)",
```

Other integration saved views do not mention they are integrations in the name. 
```suggestion
  "name": "Security Monitoring",
```